backend/src/api/helpers.ts

```diff
@@ -16,6 +16,10 @@ export const validateBody = async <T>(
   validateOptions?: ValidationOptions
 ): Promise<T> => {
   const raw: any = plainToClass(obj, JSON.parse(body ?? '{}'));
+  // The organization parent is passed back as an organization instead of the string ID. This checks for it.
```

Comment:
Why do we need this check here? This is a general function that probably shouldn't contain super-specific code unless we really need it here. Can't we just do this check in the specific endpoints that need the parent id?